src/engine/controls/bpmcontrol.cpp

```diff
@@ -470,6 +470,11 @@ double BpmControl::calcSyncAdjustment(bool userTweakingSync) {
     double syncTargetBeatDistance = m_dSyncTargetBeatDistance.getValue();
     // We want the untweaked beat distance, so we have to add the offset here.
     double thisBeatDistance = m_pThisBeatDistance->get() + m_dUserOffset.getValue();
+    // If we are moving backwards, we have to invert the calculations.
+    if (reversed) {
+        syncTargetBeatDistance = 1.0 - syncTargetBeatDistance;
```

Comment:
This needs to be applied if the sync target plays reverse, not the follower.

Reply:
yeah.  removing